### PR TITLE
feat: show y Axis and corresponding grid lines for Metrics tab in Services/API Endpoints/Backend

### PR DESCRIPTION
**Pull request by jaywalker21**
## Description
- For current **Services** view, if we check `Overview` tab, all the widgets and graphs have their Y Axis value, but for `Metrics` tab, the Y Axis value and grid lines are not shown, which enforces users to hover over the graph to get the corresponding Y Axis values. The same also happens for **API Endpoints** and **Backends** view.
- This PR adds the Y Axis to the respective charts to make the view more useful for user.

### Testing
- Did manual testing to verify that the application is working fine.

### Checklist:
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

Below are some images for before | after comparison.

#### Before
image
image

#### After
image
image

### CODE REVIEW

File: projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts [hunk 1]
```diff
@@ -201,6 +201,13 @@ export const serviceMetricsDashboard: DashboardDefaultConfiguration = {
             'color-palette': RED_COLOR_PALETTE,
             'selectable-interval': true,
             'legend-position': LegendPosition.TopLeft,
+            'show-y-axis': true,
+            'y-axis': {
+              type: 'cartesian-axis',
+              'show-grid-lines': true,
+              'min-upper-limit': 25
+            },
+            'max-series-data-points': 150,
```
Comment (aaron-steinfeld) [inline projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts:210]: This doesn't appear related, should remain unset/auto.

**Review by aaron-steinfeld — COMMENTED** [review]
Can you verify the other metric dashboards (API, backend) are aligned with this change?

File: projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts (hunk 1 above)
Comment (jaywalker21) [reply projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts:210]: Thanks Aaron, I will remove it from both the places.

**Comment by jaywalker21** [comment]
@aaron-steinfeld I missed doing these changes for API Endpoints and Backends view.
I will push those and then ask for re-review. Thanks for pointing out.

File: projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts (hunk 1 above)
Comment (jaywalker21) [reply projects/observability/src/pages/apis/service-detail/metrics/service-metrics.dashboard.ts]: @aaron-steinfeld Have fixed this, please re-review whenever feasible.

**Review by aaron-steinfeld — APPROVED** [review]
